src/components/GrowerDetail.js

```diff
@@ -115,6 +154,37 @@ const GrowerDetail = (props) => {
     // eslint-disable-next-line
   }, [growerId, growerContext.growers]);
 
+  useEffect(() => {
+    async function loadCaptures() {
+      if (growerId) {
+				setLoading(true);
+        let filter = new FilterModel();
+        filter.planterId = growerId;
+        const captures = await capturesContext.getAllCaptures({ filter });
```

Comment:
This is an extremely inefficient way of querying counts.
You don't need to get the capture records from the API, just the capture counts (see `treeTrackerApi:getCaptureCount()`.
You can do one query per verification state, setting the active/approved values as necessary.